lib/_stream_readable.js

```diff
@@ -488,7 +488,7 @@ function onEofChunk(stream, state) {
   }
   state.ended = true;
 
-  if (state.sync && state.length) {
+  if (state.sync) {
     // if we are sync and have data in the buffer, wait until next tick
```

Comment:
prob wanna remove the "and have data in the buffer" part of the comment now

Reply:
done